staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -92,6 +97,8 @@ type AggregateCheckResult struct {
 	// ForbiddenDetails is a slice of the forbidden details from all the forbidden checks. It may include empty strings.
 	// ForbiddenReasons and ForbiddenDetails must have the same number of elements, and the indexes are for the same check.
 	ForbiddenDetails []string
+	// ErrLists is a slice of the forbidden errors from all the forbidden checks.
```

Comment:
```suggestion
	// ErrLists is a slice of the field errors from all the forbidden checks.
	// ForbiddenReasons, ForbiddenDetails and ErrLists must have the same number of elements, and the indexes are for the same check.
```

Please update other comments above accordingly.